NanoAnalysis/python/nanoZZ4lAnalysis.py

```diff
@@ -211,12 +223,13 @@
         from ZZAnalysis.NanoAnalysis.genFiller import *
         from ZZAnalysis.NanoAnalysis.cloneBranches import *
         pre_sequence = [puWeight(LEPTON_SETUP, DATA_TAG),
-                        weights,
+                        weights, 
+                        LHEFiller(),
```

Comment:
As LHEFiller relies on the additional variables that we added in v15, you should add a protection here as mentioned above, otherwise it will crash for everybody running on the current samples.
You can check the value of NANOVERSION for this purpose.